src/PostgREST/DbStructure.hs

```diff
@@ -747,3 +753,17 @@ getPgVersion = H.query () $ H.statement sql HE.unit versionRow False
   where
     sql = "SELECT current_setting('server_version_num')::integer, current_setting('server_version')"
     versionRow = HD.singleRow $ PgVersion <$> HD.value HD.int4 <*> HD.value HD.text
+
+fillSessionWithSettings :: [(Text, Text)] -> H.Session ()
+fillSessionWithSettings settings =
+    -- Send all of the config settings to the set_config function, using pgsql's `unnest` to transform arrays of values
+    H.query settings $ H.statement "SELECT set_config(unnest($1), unnest($2), false)" encoder HD.unit False
```

Comment:
Wow that's clever.

Reply:
…maybe too clever actually. Look at the logs from CI, they repeat this over and over:
```json
{
  "hint":null,
  "details":null,
  "code":"0A000",
  "message":"functions and operators can take at most one set argument"
}
```

Reply:
That's odd... it's working for me locally.

Reply:
Maybe I'm misdiagnosing the cause of that message. The tests are getting stuck though.

Reply:
No, I think you're right. I was seeing the postgresql log and believing it to be successful:

```
2018-02-12 17:46:36.836 MST [18209] LOG:  statement: SET client_encoding = 'UTF8';SET client_min_messages TO WARNING;
2018-02-12 17:46:36.839 MST [18209] LOG:  statement: SELECT 1
2018-02-12 17:46:36.845 MST [18209] LOG:  execute <unnamed>: SELECT set_config(unnest($1), unnest($2), false)
2018-02-12 17:46:36.845 MST [18209] DETAIL:  parameters: $1 = '{app.settings.test,app.settings.test2}', $2 = '{123/hi,/Users/duane/ok}'
```

But it isn't setting the values.